backend/src/main/resources/db/migration/V1__initializing_data.sql

```diff
@@ -0,0 +1,53 @@
+CREATE SCHEMA IF NOT EXISTS nrfc;
+
+CREATE TABLE IF NOT EXISTS nrfc.client_type_code (
```

Comment:
could we maintain case , all lower?

Reply:
We can, but I prefer to keep SQL instructions as uppercase based on DX ( developer experience). It is easier to read and identify where the statement ends and where the specific parameter begins.